identifiers/country-us/state-ca-local_gov.csv

```diff
@@ -497,3 +497,64 @@ ocd-division/country:us/state:ca/place:visalia/council_district:2,Visalia CA cit
+ocd-division/country:us/state:ca/place:fontana/council_district:3,Fontana City Council Member District 3
+ocd-division/country:us/state:ca/place:fontana/council_district:4,Fontana City Council Member District 4
+ocd-division/country:us/state:ca/place:fremont/council_district:1,Fremont City Council Member District 1
+ocd-division/country:us/state:ca/place:fremont/council_district:2,Fremont City Council Member District 2
+ocd-division/country:us/state:ca/place:fremont/council_district:3,Fremont City Council Member District 3
+ocd-division/country:us/state:ca/place:fremont/council_district:4,Fremont City Council Member District 4
+ocd-division/country:us/state:ca/place:fremont/council_district:5,Fremont City Council Member District 5
+ocd-division/country:us/state:ca/place:fremont/council_district:6,Fremont City Council Member District 6
+ocd-division/country:us/state:ca/place:fullerton/council_district:1,Fullerton City Council Member District 1
+ocd-division/country:us/state:ca/place:fullerton/council_district:2,Fullerton City Council Member District 2
+ocd-division/country:us/state:ca/place:fullerton/council_district:3,Fullerton City Council Member District 3
+ocd-division/country:us/state:ca/place:fullerton/council_district:4,Fullerton City Council Member District 4
+ocd-division/country:us/state:ca/place:fullerton/council_district:5,Fullerton City Council Member District 5
+ocd-division/country:us/state:ca/place:garden_grove/council_district:1,Garden Grove City Council Member District 1
+ocd-division/country:us/state:ca/place:garden_grove/council_district:2,Garden Grove City Council Member District 2
+ocd-division/country:us/state:ca/place:garden_grove/council_district:3,Garden Grove City Council Member District 3
+ocd-division/country:us/state:ca/place:garden_grove/council_district:4,Garden Grove City Council Member District 4
+ocd-division/country:us/state:ca/place:garden_grove/council_district:5,Garden Grove City Council Member District 5
+ocd-division/country:us/state:ca/place:garden_grove/council_district:6,Garden Grove City Council Member District 6
+ocd-division/country:us/state:ca/place:oceanside/council_district:1,Oceanside City Council Member District 1
+ocd-division/country:us/state:ca/place:oceanside/council_district:2,Oceanside City Council Member District 2
+ocd-division/country:us/state:ca/place:oceanside/council_district:3,Oceanside City Council Member District 3
+ocd-division/country:us/state:ca/place:oceanside/council_district:4,Oceanside City Council Member District 4
+ocd-division/country:us/state:ca/place:orange/council_district:1,Orange City Council Member District 1
+ocd-division/country:us/state:ca/place:orange/council_district:2,Orange City Council Member District 2
+ocd-division/country:us/state:ca/place:orange/council_district:3,Orange City Council Member District 3
+ocd-division/country:us/state:ca/place:orange/council_district:5,Orange City Council Member District 5
```

Comment:
Looks like district 4 and 6 are missing for Orange, CA

Reply:
added districts 4 & 6